docs/source/api/env.rst

```diff
@@ -12,8 +12,8 @@ simulator Malmo which enables **synchronous**, **stable**, and
 MineRLEnv
 ===========
 
-.. automodule:: minerl.env.core
-    :members: MineRLEnv
+.. automodule:: minerl.env._multiagent
```

Comment:
There is also _singleagent env - do we want Multi-agent support to be the default here?

Reply:
Right, I was worried that it would not have enough docs, but generating it it seems to offer description of the basic step/reset funcs, so that should be enough :D